vsphere/README.md

```diff
@@ -40,6 +40,21 @@ Depending of the `collection_level` value you set in your check configuration, n
 
 See [metadata.csv][9] for a list of metrics provided by this check.
 
+#### Collecting per-instance metrics
+
+Note that the vSphere integration has the ability to collect both per-resource metrics (such as those related to CPUs), and per-instance metrics (such as those related to CPU cores). As such, there are metrics that are only per-resource, only per-instance, or both. 
```

Comment:
We should try to add some details about what is an "instance" here and ideally point to some vSphere docs for a better understanding.

Reply:
```suggestion
**Note**: The vSphere integration has the ability to collect both per-resource metrics (such as those related to CPUs), and per-instance metrics (such as those related to CPU cores). As such, there are metrics that are only per-resource, per-instance, or both. 
```